Plugin/Magento/MysqlMq/Model/ResourceModel/Queue.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+declare(strict_types=1);
+
+namespace StoreKeeper\StoreKeeper\Plugin\Magento\MysqlMq\Model\ResourceModel;
+
+use Magento\Framework\Stdlib\DateTime\TimezoneInterface;
+use Magento\MysqlMq\Model\ResourceModel\Queue as subjectQueue;
+use Magento\MysqlMq\Model\ResourceModel\MessageCollectionFactory;
+use Magento\MysqlMq\Model\ResourceModel\MessageStatusCollectionFactory;
+use StoreKeeper\StoreKeeper\Api\Data\TaskLogInterfaceFactory;
+use StoreKeeper\StoreKeeper\Api\TaskLogRepositoryInterface;
+use StoreKeeper\StoreKeeper\Model\Consumer as eventConsumer;
+use StoreKeeper\StoreKeeper\Model\Export\AbstractExportManager;
+use StoreKeeper\StoreKeeper\Model\OrderSync\Consumer as orderConsumer;
+
+class Queue
```

Comment:
looks like you implementing as  template, is there any interface we should extent?